Src/WitsmlExplorer.Frontend/components/ContentViews/FluidsView.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { useContext, useEffect, useState } from "react";
+import NavigationContext from "../../contexts/navigationContext";
+import OperationContext from "../../contexts/operationContext";
+import OperationType from "../../contexts/operationType";
+import { ComponentType } from "../../models/componentType";
+import Fluid from "../../models/fluid";
+import FluidsReport from "../../models/fluidsReport";
+import { measureToString } from "../../models/measure";
+import ComponentService from "../../services/componentService";
+import { getContextMenuPosition } from "../ContextMenus/ContextMenu";
+import FluidContextMenu, { FluidContextMenuProps } from "../ContextMenus/FluidContextMenu";
+import { ContentTable, ContentTableColumn, ContentTableRow, ContentType } from "./table";
+
+export interface FluidsRow extends ContentTableRow {
```

Comment:
It is possible to make the interface less verbose, if desired, as follows (also notice that 'export' is removed as the interface is not used outside the file):
```
type AllStrings<Type> = {
  [Property in keyof Type as Exclude<Property, "rheometers">]: string;
};

interface FluidsRow extends ContentTableRow, AllStrings<Fluid> {
  fluid: Fluid;
}
```